xml/System.Security.Cryptography/HMACMD5.xml

```diff
@@ -482,12 +490,24 @@
         <Parameter Name="destination" Type="System.Span&lt;System.Byte&gt;" Index="2" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">The HMAC key.</param>
+        <param name="source">The stream to HMAC.</param>
+        <param name="destination">The buffer to receive the HMAC value.</param>
+        <summary>Computes the HMAC of a stream using the MD5 algorithm.</summary>
+        <returns>The total number of bytes written to <paramref name="destination" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="source" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentException">
+          <p>
```

Comment:
All the `<p>` tags should be changed to `<para>`, which is a valid XMLdoc tag. `<p>` is specific to HTML - see https://github.com/dotnet/runtime/pull/66180#discussion_r819797610.

Reply:
Opened an issue to automatically do this when porting: https://github.com/dotnet/api-docs-sync/issues/115